jest.config.ts

```diff
@@ -7,6 +7,6 @@ export default async (): Promise<Config.InitialOptions> => {
     collectCoverage: true,
     testEnvironment: 'node',
     coverageReporters: ['text-summary'],
-    collectCoverageFrom: ['src/controllers/**'],
+    collectCoverageFrom: ['src/controllers/**', 'src/modules/**'],
```

Comment:
why not 'src/**' ? 
every folder you need to add here

Reply:
not everything:
config folder doesnt need tests - therefore should be in the coverage

same for entities and models, seeds, types etc...

I'm covering only the files I need to test - then if its not 100% I can actually see what I missed and add test / decide not to test it for some reason.

if ill cover everything the % wont tell me anything...